lib/components/user/trip-basics-pane.js

```diff
@@ -31,73 +37,140 @@ const StyledLabel = styled.label`
+        if (!itineraryCheckResult.data[day]) {
+          setFieldValue(day, false)
         }
+      })
+    }
+  }
+
+  render () {
+    const { errors, isCreating, touched, values: monitoredTrip } = this.props
+    const { itinerary } = monitoredTrip
+    const { itineraryCheckResult } = this.state
+
+    if (!itinerary) {
+      return <div>No itinerary to display.</div>
+    } else {
+      // Show an error indicaton when monitoredTrip.tripName is not blank (from the form's validation schema)
+      // and that tripName is not already used.
+      let tripNameValidationState = null
+      if (touched.tripName) {
+        tripNameValidationState = errors.tripName ? 'error' : null
       }
-    })
-
-    return (
-      <div>
-        <ControlLabel>Selected itinerary:</ControlLabel>
-        <TripSummary monitoredTrip={monitoredTrip} />
-
-        <FormGroup validationState={tripNameValidationState}>
-          <ControlLabel>Please provide a name for this trip:</ControlLabel>
-          {/* onBlur, onChange, and value are passed automatically. */}
-          <Field as={FormControl} name='tripName' />
-          <FormControl.Feedback />
-          {errors.tripName && <HelpBlock>{errors.tripName}</HelpBlock>}
-        </FormGroup>
-
-        <FormGroup validationState={monitoredDaysValidationState}>
-          <ControlLabel>What days to you take this trip?</ControlLabel>
-          <div>
-            {allDays.map(({ name, text }, index) => (
-              <StyledLabel className={monitoredTrip[name] ? 'bg-primary' : ''} key={index}>
-                <span>{text}</span>
-                <Field
-                  name={name}
-                  type='checkbox'
-                />
-              </StyledLabel>
-            ))}
-          </div>
-          {monitoredDaysValidationState && <HelpBlock>Please select at least one day to monitor.</HelpBlock>}
-        </FormGroup>
-      </div>
-    )
+
+      // Show a combined error indicaton when no day is selected.
+      let monitoredDaysValidationState = null
+      ALL_DAYS.forEach(day => {
+        if (touched[day]) {
+          if (!monitoredDaysValidationState) {
+            monitoredDaysValidationState = errors[day] ? 'error' : null
+          }
+        }
+      })
+
+      return (
+        <div>
+          <ControlLabel>Selected itinerary:</ControlLabel>
+          <TripSummary monitoredTrip={monitoredTrip} />
+
+          <FormGroup validationState={tripNameValidationState}>
+            <ControlLabel>Please provide a name for this trip:</ControlLabel>
+            {/* onBlur, onChange, and value are passed automatically. */}
+            <Field as={FormControl} name='tripName' />
+            <FormControl.Feedback />
+            {errors.tripName && <HelpBlock>{errors.tripName}</HelpBlock>}
+          </FormGroup>
+
+          <FormGroup validationState={monitoredDaysValidationState}>
+            <ControlLabel>What days to you take this trip?</ControlLabel>
```

Comment:
typo: `What days to you take this trip?`

Reply:
Thanks, fixed in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/38b22499b883d6ce22d87c8822d2eacb21eb668b.